apps/web/src/components/collectibles/collectible-browser.module.css

```diff
@@ -54,3 +54,51 @@
 .bold {
   @apply font-bold;
 }
+
+.flipButton {
+  display: block;
```

Comment:
Will need to "translate" these to use [Tailwind](https://tailwindcss.com/docs) with the exception of values that are not common or already defined. I recommend getting the [VS Code extension for Tailwind](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss).

Reply:
ah, I don't think I understood the spirit of tailwind. I'll take a swing. Excited to learn a new thing. 

That rhymed.